roles/common/files/etc_cron-monthly_letsencrypt-renew

```diff
@@ -1,22 +1,10 @@
 #!/bin/bash
 set -o errexit
 # Renew all live certificates with LetsEncrypt.  This needs to run at least
-# once every three months.
+# once every three months, but recommanded frequency is once a day.
```

Comment:
Spelling error.  Also, wouldn't you want to restate this in your subsequent commit?